frontend/src/components/Urgent/index.js

```diff
@@ -0,0 +1,103 @@
+import "./index.css";
+import "../index.css";
+import { useEffect, useState } from "react";
+import Radio from "@mui/material/Radio";
+import RadioGroup from "@mui/material/RadioGroup";
+import FormControlLabel from "@mui/material/FormControlLabel";
+import FormControl from "@mui/material/FormControl";
+import { useDispatch, useSelector } from "react-redux";
+import Divider from "../Divider";
+import {
+  ADD_RESPONSE,
+  REMOVE_SINGLE_RESPONSE,
+} from "../../redux/actions/formActions";
+import uuid from "react-uuid";
+
+function Urgent({ question }) {
+  const dispatch = useDispatch();
+  const [options, setOptions] = useState([]);
+  const answerList = useSelector((state) => state.questionReducer.answerList);
+
+  useEffect(() => {
+    var optionList = answerList[question.question_id ?? ""] ?? [];
+    optionList = optionList.sort((a, b) => {
+      let fa = a.answer;
+      let fb = b.answer;
+      if (fa < fb) {
+        return -1;
+      }
+      if (fa > fb) {
+        return 1;
+      }
+      return 0;
+    });
+    optionList = optionList.filter((option) => option !== "");
+    optionList.unshift({question_id: question.question_id, 
+                        fk_form_id: question.fk_form_id, 
+                        question_type: 'urgent', 
+                        question: 'Is this request urgent?', 
+                        answer_id: uuid(), 
+                        answer: "No"});
```

Comment:
We should probably use true or false here, but I think the answer is text typed, if that's the case, we should probably modify how we check the answer below.